docs/reference/ingest/ingest-node.asciidoc

```diff
@@ -1166,6 +1166,10 @@ In this example, if the `remove` processor does fail, then
 the array elements that have been processed thus far will
 be updated.
 
+Another advanced example can be found in {plugins}/using-ingest-attachment.html[attachment processor documentation].
```

Comment:
Can you change the link to `{plugins}/ingest-attachment-with-arrays.html` so people are redirected to right page automatically?